source/material_model/grain_size.cc

```diff
@@ -176,7 +176,7 @@ namespace aspect
                         ++n_T;
                       }
                   }
-                if (std::fabs(pressures[q] - pressures[p]) > get_pT_steps()[0] / 4.0)
+                if (std::fabs(pressures[q] - pressures[p]) > 1)
```

Comment:
Do you want to scale this with the pressure too, in case people do exoplanets or some other weird things where the pressure is a different order of magnitude?

Reply:
No, that would not work, and was actually the bug I was looking for. Pressure might be zero, so the equation `something_small - the_same_small > 0.0` can be true due to floating point math. I could try to come up with something complicated like `... > pressures[q] * 1e-12 + epsilon`, would you prefer that?

Reply:
I am not sure. I'll let you decide what makes more sense.